DFC.ServiceTaxonomy.Editor/Recipes/01_SkillsToolkit_Pages.recipe.json

```diff
@@ -0,0 +1,334 @@
+{
+    "name": "01_SkillsToolkit_Pages",
+    "displayName": "01_SkillsToolkit_Pages",
+    "description": "",
+    "author": "",
+    "website": "",
+    "version": "",
+    "issetuprecipe": false,
+    "categories": [],
+    "tags": [],
+    "steps": [
+        {
+            "name": "ContentNoCache",
```

Comment:
Would you mind changing this to `Content`, so that the recipe can be re-imported?